lib/Pagarme/PagarMe.php

```diff
@@ -3,7 +3,7 @@
 abstract class PagarMe {
 	public static $api_key;
 	const live = 1;
-	const endpoint = "https://api.pagar.me";
+	const endpoint = "https://api.pagarmepci.net";
```

Comment:
This URL is not official and shouldn't be used anymore!

Reply:
https://stackoverflow.com/a/12942331

Reply:
@murilohns could you please check this info?

Reply:
I think I got it! Amazon is a root Certificate Authority. To check it:

```
openssl s_client -showcerts -servername api.pagar.me -connect api.pagar.me:443 2>/dev/null | openssl x509 -inform pem -noout -text
```

This library bundles [(here)](https://github.com/pagarme/pagarme-php/blob/V2/lib/Pagarme/ca-certificates.crt) the trust chain it uses to validate certificates, and it looks like the last update of the `ca-certificates` chain was made back in 2014.